.cirrus.yml

```diff
@@ -642,22 +706,18 @@ arm64_container_image_docker_builder:
   env:
     CIRRUS_ARCH: arm64
   << : *DOCKER_BUILD_TEMPLATE
-  << : *SKIP_TASK_ON_PR
+  << : *BUILDS_ONLY_IF_RELEASE_TAG_NIGHTLY
 
 amd64_container_image_docker_builder:
   env:
     CIRRUS_ARCH: amd64
   << : *DOCKER_BUILD_TEMPLATE
-  << : *SKIP_CLUSTER_TEST_ON_PR
+  << : *BUILDS_ONLY_IF_PR_MASTER_RELEASE_NIGHTLY
+  << : *SKIP_IF_PR_NOT_FULL_OR_CLUSTER_TEST
 
 container_image_manifest_docker_builder:
   cpu: 1
-  # Push master builds to zeek/zeek-dev, or tagged release branches to zeek/zeek
-  only_if: >
-    ( $CIRRUS_CRON == '' ) &&
-    ( $CIRRUS_REPO_FULL_NAME == 'zeek/zeek' &&
-      ( $CIRRUS_BRANCH == 'master' ||
-        $CIRRUS_TAG =~ 'v[0-9]+\.[0-9]+\.[0-9]+(-rc[0-9]+)?$' ) )
+  << : *BUILDS_ONLY_IF_RELEASE_TAG_NIGHTLY
```

Comment:
Certainly for a follow-up: "Nightly if any changes since last time" would be really nice so we don't push container images every night when there anyhow were no changes. I think Cirrus CI can do that...